shaka-lab-node/linux/debian/shaka-lab-node.postinst

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Post-install script.  Runs as root after package installation.
+
+# Fail on error
+set -e
+
+# Create a non-root user for the service.
+adduser \
+  --quiet \
+  --system \
+  shaka-lab-node \
+  --home /opt/shaka-lab/shaka-lab-node \
+  --no-create-home
+
+# Grant that user access to docker, to be able to run the Tizen node.
+# Don't fail if docker is not installed.
+usermod -a -G docker shaka-lab-node || true
+
+# Install NVM, which we don't / shouldn't bundle into the deb package.
```

Comment:
It's disappointing to have to do this, but even the LTS Ubuntu release from earlier this year only ships Node v12, which was already 2 years old when that version of Ubuntu was released.  So even with this year's Ubuntu LTS, we can't depend on a packaged version of nodejs.  And we also don't want to limit this package to fresh versions of Ubuntu.  So NVM seems like a good escape hatch.

I'm open to other options, though, if you have any suggestions.

Reply:
No worries.  Just putting it out there in recognition of the compromise I had to make on Linux.